content/id/docs/reference/glossary/service.md

```diff
@@ -2,22 +2,22 @@
 title: Service
 id: service
 date: 2020-04-05
-full_link: /docs/concepts/services-networking/service/
+full_link: /id/docs/concepts/services-networking/service/
 short_description: >
-  Sebuah Cara untuk mengekspos aplikasi yang berjalan pada sebuah kumpulan Pod sebagai layanan jaringan.
+  Cara untuk mengekspos aplikasi yang berjalan pada satu set Pod sebagai layanan jaringan.
```

Comment:
"set" = "kumpulan"
```suggestion
  Cara untuk mengekspos aplikasi yang berjalan pada satu kumpulan Pod sebagai layanan jaringan.
```

Reply:
For set just make sure we’re consistent on whether using set or kumpulan here